geometric/params.py

```diff
@@ -64,7 +66,7 @@ def __init__(self, **kwargs):
         # Because TS optimization is experimental, use conservative trust radii
         self.trust = kwargs.get('trust', 0.01 if self.transition else 0.1)
         # Maximum value of trust radius
-        self.tmax = kwargs.get('tmax', 0.03 if self.transition else 0.3)
+        self.tmax = kwargs.get('tmax', 0.03 if self.transition else self.trust if self.irc else 0.3)
```

Comment:
Is this because tmax should be equal to trust in the IRC approach?

Reply:
Yes, since it will vary the stepsize based on the step quality, I set the `tmax` equal to `trust` so that the stepsize won't exceed the `trust`.